packages/hash/frontend/src/components/hooks/useAccountPages.ts

```diff
@@ -28,8 +31,20 @@ export const useAccountPages = (ownedById: string): AccountPagesInfo => {
     variables: { ownedById },
   });
 
+  const { users } = useUsers();
+  const { orgs } = useOrgs();
+
+  const ownerShortname = useMemo(
+    () =>
+      (
+        orgs?.find(({ orgAccountId }) => orgAccountId === ownedById) ??
+        users?.find(({ userAccountId }) => userAccountId === ownedById)
+      )?.shortname,
+    [users, orgs, ownedById],
+  );
```

Comment:
This seems like something we'd do in more places to get the shortname from an `OwnedById` (`AccountId`). Should this be a hook so we can swap out the implementation down the line?

Reply:
Have added a `useWorkspaceShortnameByAccountId` in https://github.com/hashintel/hash/pull/1650/commits/aa68a29649c1a34d38f273c0aa806ca1de230106